engine/access/rpc/backend/backend_block_details.go

```diff
@@ -49,11 +49,11 @@ func (b *backendBlockDetails) GetLatestBlock(_ context.Context, isSealed bool) (
 		return nil, flow.BlockStatusUnknown, status.Errorf(codes.Internal, "could not get latest block: %v", err)
 	}
 
-	status, err := b.getBlockStatus(block)
+	stat, err := b.getBlockStatus(block)
```

Comment:
why rename this? `stat` is more ambiguous than the previous value

Reply:
status variable collides with the imported package.